data/mods/Xedra_Evolved/items/inventor/bots.json

```
@@ -0,0 +1,56 @@
+[
+  {
+    "id": "bot_sniperbot",
+    "type": "TOOL",
+    "name": { "str": "inactive sniper bot" },
+    "description": "Compressed sniper century.  Ludicrious, physical law violating compression allow to fit them into small metal cylinder, but the weight compression is not as effective.",
```

```suggestion
    "description": "Compressed sniper sentry.  Ludicrious, physical law violating compression allow to fit them into small metal cylinder, but the weight compression is not as effective.",
```  
I think you mean sentry but you might have meant centurion?